src/can.rs

```diff
@@ -1,9 +1,29 @@
+//!# CAN Controller device
+//!
+//!```
+//! use mcp2517::can::Controller;
+//! use mcp2517::config::Configuration;
+//! use mcp2517::example::*;
+//!
```

Comment:
For keeping the specific examples shorter, it's common practice to hide some less important lines like the use statements.
```suggestion
//!# use mcp2517::can::Controller;
//!# use mcp2517::config::Configuration;
//!# use mcp2517::example::*;
//!#
```